integration-tests/test_status.py

```diff
@@ -2,7 +2,7 @@
 import pytest
 from time import sleep
 
-pytestmark = pytest.mark.usefixtures("register_subman")
+pytestmark = pytest.mark.usefixtures("install_katello_rpm", "register_subman")
```

Comment:
rather than this way, I'd add the `install_katello_rpm` fixture to the `register_subman` one: this way it is used automatically every time `register_subman` is used